bitnami/kafka/templates/kafka-provisioning.yaml

```diff
@@ -0,0 +1,109 @@
+{{- if .Values.provisioning.enabled }}
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: {{ include "kafka.provisioning.fullname" . }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: kafka
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- if .Values.provisioning.runAsHook }}
```

Comment:
I have tested this `if` and I would say it is not really useful. That is, whether the annotations are present or not, the job installation will behave the same one when installing the chart with `helm template`. I think we can drop this `if` and leave it as it was.